configuration/analyzer_config.json

```diff
@@ -1878,6 +1878,31 @@
     "secrets": {},
     "params": {}
   },
+  "Phishstats": {
+    "type": "observable",
+    "python_module": "phishstats.PhishStats",
+    "description": "check if the observable is a phishing site",
+    "disabled": false,
+    "external_service": true,
+    "leaks_info": true,
+    "observable_supported": [
+      "ip"
```

Comment:
phishstats can analyse ip, url and domain.

Reply:
I'll add this up in upcoming commits

Reply:
Added